addons/ui/org.openhab.ui.cometvisu/src/main/java/org/openhab/ui/cometvisu/backend/WriteResource.java

```diff
@@ -68,7 +69,7 @@ public Response setState(@Context HttpHeaders headers, @QueryParam("a") String i
             }
             return Response.ok(bean, MediaType.APPLICATION_JSON).build();
         } catch (ItemNotFoundException e) {
-            logger.error(e.getLocalizedMessage());
+            logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace